homeassistant/components/smarttub/climate.py

```diff
@@ -32,6 +37,19 @@ async def async_setup_entry(hass, entry, async_add_entities):
 class SmartTubThermostat(SmartTubEntity, ClimateEntity):
     """The target water temperature for the spa."""
 
+    PRESET_MODES = {
```

Comment:
These could be module level constants instead.

Reply:
Was addressed here https://github.com/home-assistant/core/pull/47190